src/Orleans.Core/Configuration/CollectionAgeLimitConstants.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Orleans.Configuration
+{
+    internal class CollectionAgeLimitConstants
+    {
+        public static readonly double DefaultCollectionAgeLimitInMinutes = 120;
```

Comment:
I prefer a default of zero (i.e, remove this and make `Minutes` an auto-property) - we can let the config validator throw when no value is set.